### ISSUE TITLE
[Enhancement]: Showing a loader when clicking on a button.

### ISSUE THREAD

**Issue by biswarupgh0sh** [woothemes/woocommerce#52508]
### Describe the solution you'd like

I an thinking of a feature that would be such as, when user will click on button a loader will appear to show that backend is working, for example a loader can be added to when the user will click on "Create an account" or "Login" button.

### Describe alternatives you've considered

Otherwise, an animation on the screen that will appear once the button will be clicked referring that things are being loaded.

**Comment by coreymckrill** [comment]
@biswarupgh0sh Are there specific views or actions that you're thinking of? Many things in WooCommerce already have loading animations...

**Comment by biswarupgh0sh** [comment]
> @biswarupgh0sh Are there specific views or actions that you're thinking of? Many things in WooCommerce already have loading animations...


Yes, actually when we visit the site, and we click on Get Started or Login, then once we click on the button it blinks, but I am thinking to implement a loader which will rotate to showcase that the backend is working on the request. And i can also change the button color to "bg-gray-400" or something.

**Comment by gigitux** [comment]
Thanks for taking the time to leave your feature request. While we appreciate you sharing your idea with us, it doesn’t fit in with our current priorities for the project. This doesn’t mean what you are suggesting isn’t a good idea or doesn’t make sense – it’s just that we have limited time to work on various projects. At some point, we may revisit our priorities and look through the list of suggestions like this one to see if it warrants a second look. Meanwhile, don’t be alarmed if you don’t see any activity on this issue for a while.
